spark/src/test/java/io/ddf/spark/etl/TransformationHandlerTest.java

```diff
@@ -132,7 +132,7 @@ public void testReservedFactor() throws DDFException {
     ddf.setAsFactor("month");
```

Comment:
No need to do in this PR, but we should have a way for users to provide the list of levels in a certain order OR a map from level to index. One example is in binary classification with two labels Yes an No, Yes may be map to 1 as the positive class and vice versa depends on users' need.